### DISCUSSION TITLE
RAML support wanted!

### DISCUSSION THREAD

**Discussion by rikuyaosawa** [nvim-treesitter/nvim-treesitter#7184] [Ideas]
I want a support for RESTful API Modeling Language (aka RAML). I have been writing RAML a lot at work, so I would love it if there is one. (I realize it's already got an impressive list of languages that are supported in nvim-treesitter. I appreciate the work you all do to make our dev experiece better and smoother!)

**Comment by clason** [comment]
Sorry, but we don't take requests. You'll have to write your own, or lobby someone within the RAML community.
